app/views/layouts/_footer.html.erb

```diff
@@ -2,7 +2,7 @@
 
 <div class="row">
 	<div class="col text-center">
-		<small><%= t 'navigation.footer.please_send_feedback' %> <%= link_to 'DCAF.CMapp@gmail.com', mailto: 'DCAF.CMapp@gmail.com' %>.</small>
+		<small><%= t 'navigation.footer.please_send_feedback' %> <%= link_to 'DCAF.CMapp@gmail.com', 'mailto:DCAF.CMapp@gmail.com' %>.</small>
```

Comment:
lol

Reply:
I think this is an artifact of my brief foray into git hell... some rebase + fast forwarding issues; should go away when I remerge main.